### DISCUSSION TITLE
Deepl报错："Too many requests"

### DISCUSSION THREAD

**Discussion by fionawang19** [pot-app/pot-desktop#259] [Q&A] [answered]
用Deepl翻译过若干次后，报如下错误：
Http Request Error
Http Status: 429
{"jsonrpc":"2.0","error":{"code":1042911,"message":"Too many requests"}}

请问大家该怎么解决呢？谢谢！
![微信截图_20230619170154](https://github.com/pot-app/pot-desktop/assets/137051255/0f6c7fbf-8cf7-449b-9fca-ce9084e796a7)

**Comment by Pylogmon** [comment, marked as answer]
你的IP请求太频繁了，关了代理就好了，如果没开代理的话，等十分钟就好了，要彻底解决的话需要填写自己的key来使用

**Reply by fionawang19** [reply]
非常感谢！等待一会儿就可以继续使用了。
